one-ui/ui/src/pages/Bench.test.tsx

```diff
@@ -1,9 +1,185 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import Bench from './Bench';
-
-describe('Bench component', () => {
-  it('should render correctly', () => {
-    shallow(<Bench />);
-  });
-});
\ No newline at end of file
+import { render, fireEvent, waitForElement, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import axiosMock from 'axios'
+import  Bench from '../pages/Bench';
+import {IRolesContextInterface, RolesContext} from '../util/roles';
+
+jest.mock('axios');
+
+class MockRolesService implements IRolesContextInterface{
+    public roles: string[] = [];
+
+    public setRoles: (roles: string[]) => void = (roles: string[]) => {
+        this.roles = roles;
+    };
+    public canReadMappings:() => boolean = () => {
+        return false;
+    };
+    public canWriteMappings:() => boolean = () => {
+        return false;
+    };
+    public canReadMatchMerge:() => boolean = () => {
+        return false;
+    };
+    public canWriteMatchMerge:() => boolean = () => {
+        return false;
+    };
+    public canReadLoadData:() => boolean = () => {
+        return true;
+    };
+    public canWriteLoadData:() => boolean = () => {
+        return true;
+    };
+    public canReadEntityModels:() => boolean = () => {
+        return false;
+    };
+    public canWriteEntityModels:() => boolean = () => {
+        return false;
+    };
+    public canReadFlows:() => boolean = () => {
+        return true;
+    };
+    public canWriteFlows:() => boolean = () => {
+        return true;
+    };
+    public canReadStepDefinitions:() => boolean = () => {
+        return false;
+    };
+    public canWriteStepDefinitions:() => boolean = () => {
+        return false;
+    };
+    public hasOperatorRole:() => boolean = () => {
+        return true;
+    };
+}
+
+//Mock Responses
```

Comment:
Mock responses below can also be reused. Can you pls move them into data.config.
Also pls change the flow/step name and uri